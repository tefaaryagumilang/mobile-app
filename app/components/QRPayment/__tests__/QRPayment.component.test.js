import React from 'react';
import renderer from 'react-test-renderer';
import QRPaymentPayment from '../QRPayment.component';
import {Provider} from 'react-redux';
import {createStore} from 'redux';
import {reduxForm} from 'redux-form';
jest.mock('react-timer-mixin');

const spy = jest.fn();
const store = createStore(() => ({}));
const DecoratedForm = reduxForm({form: 'testForm'})(QRPaymentPayment);

describe('QRPaymentPayment component', () => {
  const formValues = {
    amount: '219276',
    myAccount: {accountMode: null,
      accountNumber: '0002838796',
      accountType: 'savingaccount',
      accountTypeCode: '6003',
      allowFlag: 'ft|bp|oa|tds',
      balances: {
        accountNumber: '0002838796',
        availableBalance: 32047803.55,
        currency: 'IDR',
        currentBalance: 32047803.55,
        workingBalance: 32047803.55
      },
      bank: 'PT BANK SINARMAS',
      bankBranch: {
        address: 'Jl. K.H. Fachruddin Ruko Alfa No. |',
        code: 'ID0010007',
        id: 7,
        name: 'KC Tanah Abang',
      },
      currency: 'IDR',
      display: '0002838796 • savingaccount • M FERDIANSYAH',
      id: 10322,
      label: '0002838796/IDR/SavingAccount/M FERDIANSYAH',
      name: 'M FERDIANSYAH',
    },
    paymentMode: 'outstanding'
  };

  it('renders correctly', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <DecoratedForm handleSubmit={spy} formValues={formValues}/>
      </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
