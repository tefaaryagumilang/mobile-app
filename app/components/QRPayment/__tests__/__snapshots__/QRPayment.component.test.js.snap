// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QRPaymentPayment component renders correctly 1`] = `
<KeyboardAwareScrollView
  contentContainerStyle={
    Object {
      "alignItems": "stretch",
    }
  }
  extraHeight={120}
  keyboardShouldPersistTaps="handled"
  style={
    Array [
      Object {
        "backgroundColor": "rgba(0,0,0, 0.09)",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#FFFFFF",
        },
        Object {
          "marginVertical": 5,
        },
      ]
    }
  >
    <View
      style={
        Object {
          "padding": 30,
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 18,
            "fontWeight": "700",
            "paddingBottom": 20,
          }
        }
      >
        Untuk Pembayaran undefined
      </Text>
      <View
        style={
          Object {
            "flex": 1,
            "paddingBottom": 15,
          }
        }
      >
        <View>
          <View>
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "marginVertical": 2,
                }
              }
            >
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              >
                <Text>
                  Harga
                </Text>
              </View>
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "flex-start",
                      "fontSize": 18,
                      "marginHorizontal": 10,
                      "marginLeft": 40,
                    }
                  }
                >
                  IDR --
                </Text>
              </View>
            </View>
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "marginVertical": 2,
                }
              }
            >
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              >
                <Text>
                  Diskon
                </Text>
              </View>
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "flex-start",
                      "fontSize": 18,
                      "marginHorizontal": 10,
                      "marginLeft": 40,
                    }
                  }
                >
                  IDR --
                </Text>
              </View>
            </View>
            <View
              style={
                Object {
                  "borderTopColor": "#ccc",
                  "borderTopWidth": 1,
                  "marginVertical": 15,
                }
              }
            />
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "marginVertical": 2,
                }
              }
            >
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              >
                <Text>
                  Total
                </Text>
              </View>
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "flex-start",
                      "fontSize": 18,
                      "marginHorizontal": 10,
                      "marginLeft": 40,
                    }
                  }
                >
                  IDR --
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#FFFFFF",
        },
        Object {
          "marginVertical": 5,
        },
      ]
    }
  >
    <View
      style={
        Object {
          "padding": 30,
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 18,
            "fontWeight": "700",
            "paddingBottom": 20,
          }
        }
      >
        Rekening Anda
      </Text>
      <View>
        <View
          style={
            Array [
              Object {
                "backgroundColor": "#ffffff",
                "borderBottomWidth": 1,
                "borderColor": "#ccc",
                "height": 50,
                "justifyContent": "center",
                "marginTop": 10,
              },
              Object {},
            ]
          }
        >
          <View>
            <View
              accessible={true}
              clickable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "height": 50,
                  "justifyContent": "space-between",
                  "opacity": 1,
                }
              }
            >
              <Text
                style={
                  Array [
                    Array [
                      Object {
                        "flex": 1,
                      },
                      Object {
                        "color": "#000",
                        "fontSize": 14,
                      },
                      Object {
                        "fontWeight": "700",
                      },
                    ],
                    Object {},
                  ]
                }
              >
                Pilih akun
              </Text>
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": undefined,
                      "fontSize": 12,
                    },
                    Array [
                      Object {
                        "color": "#000000",
                        "paddingRight": 15,
                      },
                      Object {},
                    ],
                    Object {
                      "fontFamily": "simas",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
              >
                î¤†
              </Text>
            </View>
            <Modal
              animationType="none"
              hardwareAccelerated={false}
              hideOnBack={false}
              onBackButtonPress={[Function]}
              onRequestClose={[Function]}
              transparent={true}
              visible={false}
            >
              <View
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "bottom": 0,
                    "height": 1334,
                    "left": 0,
                    "opacity": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 750,
                  }
                }
              />
              <View
                hideOnBack={false}
                onBackButtonPress={[Function]}
                style={
                  Object {
                    "flex": 1,
                    "justifyContent": "center",
                    "margin": 0,
                    "padding": 0,
                  }
                }
                transparent={true}
              >
                <View
                  contentContainerStyle={
                    Object {
                      "flex": 1,
                    }
                  }
                  extraHeight={120}
                  keyboardShouldPersistTaps="handled"
                  scrollEnabled={false}
                  style={
                    Object {
                      "flex": 1,
                    }
                  }
                >
                  <View
                    accessible={true}
                    clickable={true}
                    onClick={[Function]}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Array [
                        Object {
                          "backgroundColor": "rgba(37, 8, 10, 0.78)",
                          "flex": 1,
                          "justifyContent": "center",
                          "padding": 20,
                        },
                        Object {
                          "justifyContent": "flex-end",
                          "padding": 0,
                        },
                      ]
                    }
                  >
                    <View
                      accessible={true}
                      clickable={true}
                      onClick={[Function]}
                      onResponderGrant={[Function]}
                      onResponderMove={[Function]}
                      onResponderRelease={[Function]}
                      onResponderTerminate={[Function]}
                      onResponderTerminationRequest={[Function]}
                      onStartShouldSetResponder={[Function]}
                      style={
                        Array [
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "#FFFFFF",
                            "borderRadius": 20,
                            "padding": 20,
                          },
                          Object {
                            "backgroundColor": "transparent",
                            "padding": 0,
                          },
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "#FFFFFF",
                            "borderRadius": 20,
                            "padding": 20,
                          },
                        ]
                      }
                    >
                      <View
                        style={
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "#ededed",
                            "justifyContent": "center",
                            "padding": 0,
                            "width": 750,
                          }
                        }
                      >
                        <View
                          style={
                            Object {
                              "alignItems": "center",
                              "flexDirection": "row",
                              "height": 50,
                              "justifyContent": "space-between",
                              "paddingHorizontal": 20,
                              "width": 750,
                            }
                          }
                        >
                          <View
                            style={
                              Object {
                                "flexDirection": "row",
                              }
                            }
                          >
                            <View
                              accessible={true}
                              clickable={true}
                              isTVSelectable={true}
                              onClick={[Function]}
                              onResponderGrant={[Function]}
                              onResponderMove={[Function]}
                              onResponderRelease={[Function]}
                              onResponderTerminate={[Function]}
                              onResponderTerminationRequest={[Function]}
                              onStartShouldSetResponder={[Function]}
                              style={
                                Object {
                                  "opacity": 1,
                                  "paddingRight": 20,
                                }
                              }
                            >
                              <Text
                                allowFontScaling={false}
                                style={
                                  Array [
                                    Object {
                                      "color": undefined,
                                      "fontSize": 20,
                                    },
                                    Object {
                                      "color": "#007AFF",
                                      "transform": Array [
                                        Object {
                                          "rotate": "180deg",
                                        },
                                      ],
                                    },
                                    Object {
                                      "fontFamily": "simas",
                                      "fontStyle": "normal",
                                      "fontWeight": "normal",
                                    },
                                    Object {},
                                  ]
                                }
                              >
                                î¤†
                              </Text>
                            </View>
                            <View
                              accessible={true}
                              clickable={true}
                              isTVSelectable={true}
                              onClick={[Function]}
                              onResponderGrant={[Function]}
                              onResponderMove={[Function]}
                              onResponderRelease={[Function]}
                              onResponderTerminate={[Function]}
                              onResponderTerminationRequest={[Function]}
                              onStartShouldSetResponder={[Function]}
                              style={
                                Object {
                                  "opacity": 1,
                                  "paddingLeft": 20,
                                }
                              }
                            >
                              <Text
                                allowFontScaling={false}
                                style={
                                  Array [
                                    Object {
                                      "color": undefined,
                                      "fontSize": 20,
                                    },
                                    Object {
                                      "color": "#007AFF",
                                    },
                                    Object {
                                      "fontFamily": "simas",
                                      "fontStyle": "normal",
                                      "fontWeight": "normal",
                                    },
                                    Object {},
                                  ]
                                }
                              >
                                î¤†
                              </Text>
                            </View>
                          </View>
                          <View
                            accessible={true}
                            clickable={true}
                            isTVSelectable={true}
                            onClick={[Function]}
                            onResponderGrant={[Function]}
                            onResponderMove={[Function]}
                            onResponderRelease={[Function]}
                            onResponderTerminate={[Function]}
                            onResponderTerminationRequest={[Function]}
                            onStartShouldSetResponder={[Function]}
                            style={
                              Object {
                                "opacity": 1,
                              }
                            }
                          >
                            <Text
                              style={
                                Object {
                                  "color": "#007AFF",
                                  "fontSize": 16,
                                  "fontWeight": "700",
                                }
                              }
                            >
                              Selesai
                            </Text>
                          </View>
                        </View>
                        <View>
                          <View
                            style={
                              Object {
                                "width": 750,
                              }
                            }
                          >
                            <RCTPicker
                              items={
                                Array [
                                  Object {
                                    "label": "Pilih akun",
                                    "textColor": undefined,
                                    "value": "Pilih akun",
                                  },
                                ]
                              }
                              onChange={[Function]}
                              onResponderTerminationRequest={[Function]}
                              onStartShouldSetResponder={[Function]}
                              selectedIndex={0}
                              style={
                                Array [
                                  Object {
                                    "height": 216,
                                  },
                                  Object {
                                    "backgroundColor": "#FFFFFF",
                                  },
                                ]
                              }
                            />
                          </View>
                        </View>
                      </View>
                    </View>
                  </View>
                </View>
              </View>
            </Modal>
          </View>
        </View>
      </View>
      <Text
        style={
          Array [
            Object {
              "color": "#666",
            },
            Object {
              "fontSize": 14,
            },
          ]
        }
      >
        Saldo tersedia
         : Rp 
        32.047.803.55
      </Text>
    </View>
  </View>
  <View
    style={
      Array [
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
          },
          Object {
            "marginVertical": 5,
          },
        ],
        Object {
          "padding": 30,
        },
      ]
    }
  >
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FA3F70",
              "borderRadius": 30,
              "flexDirection": "row",
              "height": 47,
              "justifyContent": "center",
              "paddingHorizontal": 15,
            },
            undefined,
            Object {},
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontSize": 16,
            }
          }
        >
          BERIKUTNYA
        </Text>
      </View>
    </View>
  </View>
  <View>
    <Modal
      animationType="none"
      hardwareAccelerated={false}
      hideOnBack={true}
      onBackButtonPress={[Function]}
      onRequestClose={[Function]}
      transparent={true}
      visible={false}
    >
      <View
        style={
          Object {
            "backgroundColor": "transparent",
            "bottom": 0,
            "height": 1334,
            "left": 0,
            "opacity": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 750,
          }
        }
      />
      <View
        hideOnBack={true}
        onBackButtonPress={[Function]}
        style={
          Object {
            "flex": 1,
            "justifyContent": "center",
            "margin": 0,
            "padding": 0,
          }
        }
        transparent={true}
      >
        <View
          contentContainerStyle={
            Object {
              "flex": 1,
            }
          }
          extraHeight={120}
          keyboardShouldPersistTaps="handled"
          scrollEnabled={false}
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View
            accessible={true}
            clickable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(37, 8, 10, 0.78)",
                  "flex": 1,
                  "justifyContent": "center",
                  "padding": 20,
                },
                Object {},
              ]
            }
          >
            <View
              accessible={true}
              clickable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#FFFFFF",
                    "borderRadius": 20,
                    "padding": 20,
                  },
                  Object {},
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#FFFFFF",
                    "borderRadius": 20,
                    "padding": 20,
                  },
                ]
              }
            >
              <View
                style={
                  Object {
                    "alignItems": "flex-start",
                    "flexDirection": "row",
                    "padding": 5,
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "padding": 5,
                      },
                      Object {
                        "color": "#000",
                        "fontSize": 24,
                      },
                      Object {
                        "textAlign": "center",
                      },
                    ]
                  }
                >
                  Masukkan EasyPIN
                </Text>
                <Image
                  source="easypin-small.png"
                  style={
                    Object {
                      "height": 22,
                      "marginLeft": 5,
                      "marginTop": 5,
                      "width": 22,
                    }
                  }
                />
              </View>
              <View
                style={
                  Object {
                    "alignItems": "flex-start",
                    "flexDirection": "row",
                    "padding": 14,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#000000",
                      "fontSize": 16,
                      "paddingLeft": 45,
                      "textAlign": "center",
                    }
                  }
                >
                  Masukkan 6 digit kode keamanan Anda
                </Text>
                <View
                  accessible={true}
                  clickable={false}
                  isTVSelectable={true}
                  onClick={[Function]}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "opacity": 1,
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "opacity": 1,
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "borderRadius": 5,
                          "height": 15,
                          "justifyContent": "center",
                          "marginRight": 40,
                          "marginTop": 15,
                          "width": 15,
                        }
                      }
                    >
                      <Image
                        source="question-grey.png"
                      />
                    </View>
                  </View>
                  <Modal
                    hardwareAccelerated={false}
                    onRequestClose={[Function]}
                    transparent={true}
                    visible={false}
                  >
                    <View
                      style={
                        Object {
                          "backgroundColor": "rgba(0,0,0,0.5)",
                          "flex": 1,
                          "opacity": 0,
                        }
                      }
                    >
                      <View
                        accessible={true}
                        clickable={true}
                        isTVSelectable={true}
                        onClick={[Function]}
                        onResponderGrant={[Function]}
                        onResponderMove={[Function]}
                        onResponderRelease={[Function]}
                        onResponderTerminate={[Function]}
                        onResponderTerminationRequest={[Function]}
                        onStartShouldSetResponder={[Function]}
                        style={
                          Object {
                            "flex": 1,
                            "opacity": 1,
                          }
                        }
                      >
                        <View
                          style={
                            Object {
                              "backgroundColor": "transparent",
                              "left": undefined,
                              "position": "absolute",
                              "top": undefined,
                              "transform": Array [
                                Object {
                                  "scale": 0,
                                },
                              ],
                            }
                          }
                        >
                          <View
                            onLayout={[Function]}
                            style={
                              Object {
                                "alignItems": "flex-start",
                                "backgroundColor": "transparent",
                              }
                            }
                          >
                            <View
                              style={
                                Array [
                                  Object {
                                    "alignSelf": "stretch",
                                    "backgroundColor": "white",
                                    "borderRadius": 5,
                                    "overflow": "hidden",
                                  },
                                  undefined,
                                ]
                              }
                            >
                              <View
                                style={
                                  Array [
                                    Object {
                                      "padding": 10,
                                    },
                                    Array [
                                      undefined,
                                    ],
                                  ]
                                }
                              >
                                <View
                                  accessible={true}
                                  clickable={true}
                                  isTVSelectable={true}
                                  onClick={[Function]}
                                  onResponderGrant={[Function]}
                                  onResponderMove={[Function]}
                                  onResponderRelease={[Function]}
                                  onResponderTerminate={[Function]}
                                  onResponderTerminationRequest={[Function]}
                                  onStartShouldSetResponder={[Function]}
                                  style={
                                    Object {
                                      "opacity": 1,
                                    }
                                  }
                                >
                                  <Text
                                    style={null}
                                  >
                                    EasyPIN adalah 6 digit angka keamanan untuk otorisasi login dan beberapa transaksi. Mohon diajaga kerahasiaan EasyPIN Anda. Kami tidak mengirimkan SMS apapun mengenai EasyPIN.
                                  </Text>
                                </View>
                              </View>
                            </View>
                            <View
                              style={
                                Array [
                                  Object {
                                    "backgroundColor": "transparent",
                                    "borderBottomColor": "transparent",
                                    "borderBottomWidth": 0,
                                    "borderLeftColor": "transparent",
                                    "borderLeftWidth": 10,
                                    "borderRightColor": "transparent",
                                    "borderRightWidth": 10,
                                    "borderStyle": "solid",
                                    "borderTopColor": "white",
                                    "borderTopWidth": 10,
                                    "height": 10,
                                    "width": 10,
                                  },
                                  Object {
                                    "left": 0,
                                    "marginLeft": 0,
                                  },
                                  null,
                                ]
                              }
                            />
                          </View>
                        </View>
                      </View>
                    </View>
                    <View
                      style={
                        Object {
                          "backgroundColor": "transparent",
                          "height": 0,
                          "left": 0,
                          "opacity": 0,
                          "position": "absolute",
                          "top": 0,
                          "transform": Array [
                            Object {
                              "scale": 1,
                            },
                          ],
                          "width": 0,
                        }
                      }
                    >
                      <View
                        accessible={true}
                        clickable={true}
                        isTVSelectable={true}
                        onClick={[Function]}
                        onResponderGrant={[Function]}
                        onResponderMove={[Function]}
                        onResponderRelease={[Function]}
                        onResponderTerminate={[Function]}
                        onResponderTerminationRequest={[Function]}
                        onStartShouldSetResponder={[Function]}
                        style={
                          Object {
                            "opacity": 1,
                          }
                        }
                      >
                        <View
                          style={
                            Object {
                              "alignItems": "center",
                              "borderRadius": 5,
                              "height": 15,
                              "justifyContent": "center",
                              "marginRight": 40,
                              "marginTop": 15,
                              "width": 15,
                            }
                          }
                        >
                          <Image
                            source="question-grey.png"
                          />
                        </View>
                      </View>
                    </View>
                  </Modal>
                </View>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "alignSelf": "center",
                      "borderBottomColor": "#FA3F70",
                      "borderBottomWidth": 1,
                      "paddingTop": 10,
                    },
                    Object {
                      "width": 150,
                    },
                  ]
                }
              >
                <TextInput
                  allowFontScaling={true}
                  autoFocus={true}
                  keyboardType="numeric"
                  maxLength={6}
                  onChangeText={[Function]}
                  rejectResponderTermination={true}
                  secureTextEntry={true}
                  style={
                    Object {
                      "fontSize": 24,
                      "height": 60,
                      "paddingVertical": 5,
                      "textAlign": "center",
                    }
                  }
                  underlineColorAndroid="transparent"
                  value=""
                />
              </View>
              <View
                style={
                  Object {
                    "alignItems": "flex-start",
                    "justifyContent": "center",
                    "padding": 5,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#ccc",
                      "fontSize": 12,
                      "paddingHorizontal": 50,
                      "textAlign": "center",
                    }
                  }
                >
                  Akun Anda akan diblokir setelah 3 kali salah mencoba
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </Modal>
  </View>
</KeyboardAwareScrollView>
`;
